.github/workflows/spec.yml

```diff
@@ -25,13 +25,13 @@ jobs:
           - mysql
           - postgresql
           - sqlite3
-        include:
-          - ruby: truffleruby-head
-            db: postgresql
-            gemfile: gemfiles/activerecord_6.1.gemfile
-          - ruby: truffleruby-head
-            db: postgresql
-            gemfile: gemfiles/activerecord_7.0.gemfile
+        # include:
+        #   - ruby: truffleruby-head
+        #     db: postgresql
+        #     gemfile: gemfiles/activerecord_6.1.gemfile
+        #   - ruby: truffleruby-head
+        #     db: postgresql
+        #     gemfile: gemfiles/activerecord_7.0.gemfile
```

Comment:
Why not just delete?

Reply:
Done in https://github.com/sharesight/acts-as-taggable-on/pull/3/commits/5cbe5a78d68f19f104f3abb6baeff4b22b9e8c27 👍🏻